namespaces/cloud-platform-live-0.k8s.integration.dsd.io/pecs-move-platform-backend-staging/resources/rds.tf

```diff
@@ -0,0 +1,24 @@
+module "rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=3.1"
+
```

Comment:
This module is now at version 4.0

Reply:
For RDS: The latest module (4.0) does not support Live-0 deployment. Be sure to use the previous one (3.1) is you need to deploy to Live-0.